helpers/addresses.py

```diff
@@ -293,6 +295,7 @@
         "bbveCVX-CVX-f": "0x0c41A8613fbeFCC8d6e5dF1020DBb336F875247F",
         "ibBTCCrv": "0xe505F7C2FFcce7Ae4b076456BC02A70D8fe8d4d2",
         "restitution": "0x3F61344BA56df00dad9bBcA05d98CA2AeC43Ba0B",
+        "badgerCrv_temp": "0xa54d8a596B4022CC4436b692C8ea0E342405eB6e"
```

Comment:
why temp? ah because governance is `deployer2`? maybe not include non prod addresses in the address book yet..?

Reply:
That's right, we are using this controller temporarily for launch but we will migrate the vault to one of the others sooner than later. I still agree that it is a good idea to keep in here for the time being as it will be a production controller. Will remove it once the migration happens.